src/Microsoft.PowerShell.Commands.Utility/commands/utility/ShowCommand/ShowCommand.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.PowerShell.Commands
     /// Show-Command displays a GUI for a cmdlet, or for all cmdlets if no specific cmdlet is specified.
     /// </summary>
     [Cmdlet(VerbsCommon.Show, "Command", HelpUri = "https://go.microsoft.com/fwlink/?LinkID=2109589")]
+    [OutputType(typeof(string))]
     public class ShowCommandCommand : PSCmdlet, IDisposable
```

Comment:
There's another call site of the `WriteObject` below, and that would be an object type, right?

```c#
        private void Output_DataAdded(object sender, DataAddedEventArgs e)
        {
            this.WriteObject(((PSDataCollection<object>)sender)[e.Index]);
        }
```

Reply:
I think the command is buggy. I can't make it run any commands, it only types the command into the prompt. If I try back in Windows PowerShell the same thing happens except it also appends an "m" to the command that it types in. It works like you would expect in ISE though.  
Even if it did work I don't think Object would be a useful type to add to the OutputTypes because it's like ForEach-Object where the output depends on what you tell it to run.

Reply:
Yeah, it seems buggy, or maybe that's expected and the `Run` only works in ISE?
In this case, maybe just undo the change in this file.

Reply:
I found `Show-Command` doesn't do streaming at all. `Show-Command | % { $_.Length }` simply stops once clicking `Run`, and the command name shows up in the prompt, meaning somehow the command name is written into stdin of the running pwsh process.
I have reverted the change in this file. Let me know if you have any concerns.